### PR TITLE
maxConcurrent must be set only once per node

### PR DESCRIPTION
**Pull request by harshavardhana**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
maxConcurrent must be set only once per node

## Motivation and Context
avoids any racy behavior

## How to test this PR?
CI/CD would test these appropriately

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/xl-storage-disk-id-check.go [hunk 1]
```diff
@@ -177,13 +177,32 @@ func (e *lockedLastMinuteLatency) total() AccElem {
...
+		if s != "" {
+			diskMaxConcurrent, _ = strconv.Atoi(s)
+		}
+	})
+
 	if diskMaxConcurrent <= 0 {
 		diskMaxConcurrent = 512
 		if storage.rotational {
-			diskMaxConcurrent = 32
+			diskMaxConcurrent = int(storage.nrRequests)/2
```
Comment (vadmeste) [inline cmd/xl-storage-disk-id-check.go]: let's do if storage.nrRequests == 0, we can set it to 32 by default. I am not sure if nrRequets is always present in /sys/

**Review by vadmeste — COMMENTED** [review]
One comment

File: cmd/xl-storage-disk-id-check.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/xl-storage-disk-id-check.go]: Oh yes I forgot to fix that.

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**